docs/guide/saved-views.md

```diff
@@ -0,0 +1,23 @@
+title: Saved Views Guide
+description: Sematext Cloud is a modern monitoring, log management, transaction tracing, and real user monitoring system that includes over 40 monitoring integrations. It is a suite of products that combine high-quality logging experience with other monitoring and alerting devops tools helping fix IT production issues
+
+Filtering and grouping data is a central part of the Sematext Cloud user interface. With Saved Views you can save your filters and groups for faster access later. If you find yourself often applying the same filters on your data then this feature can be a time saver.
+
+The feature is available on all screens where the Saved Views button is visible. This button displays "Default View" when no saved view is selected as shown in the animations below.
```

Comment:
When I saw "Saved Views button" I wondered if we are showing this "Saved Views button" in the screenshot below.  But we don't.  Well, at least there is nothing that says "Saved Views", which is what I think a person would look for.  Since the button is not actually labelled "Saved Views", maybe we need to annotate the screenshot and point to it.

Reply:
The screenshot is actually an animation that I think does a good job of showing the functionality, it's hard to miss it. Let me attach the animations here:

![SavedViewSelect](https://user-images.githubusercontent.com/126098/99546148-6ded7f80-29b6-11eb-9f1d-dfefe307e61f.gif)
![SavedViewCreate](https://user-images.githubusercontent.com/126098/99546151-6e861600-29b6-11eb-889a-1993aa8b187e.gif)

Reply:
Ahaaaaa.  Problem solved then.